### ISSUE TITLE
Add WCAG contrast level checking

### ISSUE THREAD

**Issue by fabianlindfors** [acheronfail/pixel-picker#26]
Thanks for creating Pixel Picker, it's a lovely tool! The only feature I'd like to see is a built-in contrast level checker which shows [WCAG levels](https://www.w3.org/TR/UNDERSTANDING-WCAG20/visual-audio-contrast-contrast.html).

I already have a working implementation of this which shows the contrast level in the picker, compared to the previously picked color. Here's how it currently looks: 
<img width="1169" alt="pixel-picker-wcag" src="https://user-images.githubusercontent.com/4345277/81419674-ebeee500-914e-11ea-93fb-8f3b09e0b392.png">

Let me know if you're interested in adding this feature and I'll create a PR!

**Comment by acheronfail** [comment]
Sounds like a great idea to me!

I'd like it to have a few things though:

- be able to turn it off and on via a toggle setting
- show the last picked colour when it's on

Maybe the background of the "fail" box could be the last picked colour?
Basically I just want it to be clear to the user what the last picked colour was when they have this setting enabled.

**Comment by fabianlindfors** [comment]
Glad you like it!

The background color of the "fail" box is actually the previously picked color. In the example, I first picked the title color and then hovered over the background. The green background of the "fail" box is the title color.

I'll make sure to add a toggle. Would you like this toggle to show up under a new menu item? Would you like the feature to be enabled by default?

**Comment by acheronfail** [comment]
> The background color of the "fail" box is actually the previously picked color. In the example, I first picked the title color and then hovered over the background. The green background of the "fail" box is the title color.

Excellent, glad to know we're already on the same page.

> I'll make sure to add a toggle. Would you like this toggle to show up under a new menu item? Would you like the feature to be enabled by default?

Since it's not the primary function of the app, I think it should be disabled by default. But if you save the setting like the rest of them the user only needs to enable it once and it will remain on until they turn it off.

**Comment by fabianlindfors** [comment]
Sorry that it took so long but I just got around to putting together a PR: https://github.com/acheronfail/pixel-picker/pull/28
